### PR TITLE
Unify all Chromium and Firefox based browser profiles

### PR DESCRIPTION
**Pull request by SkewedZeppelin**
See #1773

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,15 @@ Use this issue to request new profiles: [#1139](https://github.com/netblue30/fir
...
 # Current development version: 0.9.53
 
+## Browser profile unification
+
+All Chromium and Firefox browsers have been unified to instead extend
+chromium-common.profile and firefox-common.profile respectively.
+This allows for reduced maintenance and ease of adding new browsers.
+NOTE: All users of Firefox-based browsers who use addons and plugins
+that read/write from ${HOME} will need to uncomment the includes for
+firefox-common-addons.inc in firefox-common.profile.
```
Comment (smitsohu) [inline README.md]: you call it firefox-common-addons.inc here, but firefox-common-addons.profile everywhere else :)

**Comment by smitsohu** [comment]
Thanks, awesome!

The only question is if we should enable firefox-common-addons by default or not. As you say in #1773, it will probably disrupt setups that are working fine with 0.9.52 and before. 
It is only IMHO, but I would rather think the commented firefox-common-addons should be the advanced setting, and the "everything just works" uncommented firefox-common-addons the default, although it is a little less secure.

File: etc/abrowser.profile [hunk 2]
```diff
@@ -7,42 +7,14 @@ include /etc/firejail/globals.local
...
-caps.drop all
-netfilter
-nodvd
-nonewprivs
-noroot
-notv
-protocol unix,inet,inet6,netlink
-seccomp
-tracelog
+# private-etc ca-certificates,ssl,machine-id,dconf,selinux,passwd,group,hostname,hosts,localtime,nsswitch.conf,resolv.conf,xdg,gtk-2.0,gtk-3.0,X11,pango,fonts,abrowser,firefox,mime.types,mailcap,asound.conf,pulse,pki,crypto-policies
```
Comment (Fred-Barclay) [inline etc/abrowser.profile]: I'm noticing some of the firefox-based profiles have different private-etc lines (for instance, this line is different than cyberfox which is different than iceweasel). Can we maybe have a unified line in firefox-common, and then for each profile add its individual etc folders? For instance, we can probably use something like
```
private-etc ca-certificates,ssl,machine-id,dconf,selinux,passwd,group,hostname,hosts,localtime,nsswitch.conf,resolv.conf,xdg,gtk-2.0,gtk-3.0,X11,pango,fonts,mime.types,mailcap,asound.conf,pulse,pki,crypto-policies
```
in firefox-common.
Then in abrowser, just do a `private-etc abrower` before we include firefox-common.

I did a simple test and it looks like this should work.

Comment (SkewedZeppelin) [reply etc/abrowser.profile]: Will do, I wasn't sure if a second private-etc would extend or override.

File: README.md (hunk 1 above)
Comment (SkewedZeppelin) [reply README.md]: No? There is firefox-common.profile and firefox-common-addons.inc

Comment (smitsohu) [reply README.md]: Yes! Sorry

**Comment by Fred-Barclay** [comment]
Ship it!

**Comment by SkewedZeppelin** [comment]
Merged. We'll still need to decide whether to leave https://github.com/netblue30/firejail/blob/master/etc/firefox-common.profile#L9 commented or not before next release.

**Pull request merged**